README.rst

```diff
@@ -1,131 +1,156 @@
+(Chrome, Edge, etc). It can also be compiled for a number of other operating
+systems.
+
+For gameplay screenshots, see
+`our website <https://taisei-project.org/media>`__.
+
+For gameplay instructions, read `this <doc/GAME.rst>`__.
+
+For the story, read `this <doc/STORY.txt>`__. (Spoiler warning!)
+
+About Tōhō Project
+^^^^^^^^^^^^^^^^^^
+
+Tōhō Project is an indie game series (also known as "doujin" in Japanese)
+known for its ensemble cast of characters and memorable soundtracks.
+It is produced by and large by a single artist known as ZUN, and has a
+`permissive license <https://en.touhouwiki.net/wiki/Touhou_Wiki:Copyrights#Copyright_status.2FTerms_of_Use_of_the_Touhou_Project>`__
+which allows for indie derivative works such as Taisei Project to legally exist.
+
+Taisei is *not* a "clone" of Tōhō, and tells an original story with its own
+music, art, gameplay mechanics, and codebase. While some familiarity with Tōhō
+is helpful, the gameplay can be enjoyed on its own without prior knowledge of
+the series.
+
+For more information on dōjin culture,
+`click here <https://en.wikipedia.org/wiki/D%C5%8Djin>`__.
 
 Installation
 ------------
 
-Dependencies
-^^^^^^^^^^^^
+You can find precompiled binaries of the complete game on the
+`Releases <https://github.com/taisei-project/taisei/releases>`__ page on
+GitHub, available for Windows (x86/x64), Linux, and macOS.
+
+An experimental build for Nintendo Switch (homebrew) also exists (use at your
+own risk).
+
+You can also play our experimental WebGL build through your web browser
+`here <https://play.taisei-project.org/>`__. (Chromium-based browsers and
+Firefox supported.)
 
--  OpenGL >= 3.3 or OpenGL ES >= 3.0 or OpenGL ES >= 2.0 (with some extensions)
--  `cglm <https://github.com/recp/cglm>`__ >= 0.7.8
+Source Code & Development
+-------------------------
+
+Basic Dependencies
+^^^^^^^^^^^^^^^^^^
+
+Run-Time Dependencies
+_____________________
+
+Required
+********
+
+-  OpenGL >= 3.3, or OpenGL ES >= 3.0
 -  SDL2 >= 2.0.10
--  freetype2
 -  libpng >= 1.5.0
 -  libwebpdecoder >= 0.5 or libwebp >= 0.5
 -  libzip >= 1.5.0 (>= 1.7.0 recommended)
 -  libzstd >= 1.4.0
+-  freetype2
 -  opusfile
 -  zlib
 
-Optional:
+Optional
+********
 
 -  OpenSSL (for a better SHA-256 implementation; used in shader cache)
 -  SPIRV-Cross >= 2019-03-22 (for OpenGL ES backends)
 -  libshaderc (for OpenGL ES backends)
+-  `ANGLE <https://github.com/google/angle>`__ (for ANGLE support, useful for Windows builds)
 -  GameMode headers (Linux only; for automatic `GameMode
    <https://github.com/FeralInteractive/gamemode>`__ integration)
 
-Build-only dependencies
-^^^^^^^^^^^^^^^^^^^^^^^
+Build-Time Dependenices
+_______________________
 
 -  Python >= 3.6
--  meson >= 0.53.0
--  [python-zstandard](https://github.com/indygreg/python-zstandard)
+-  `python-zstandard <https://github.com/indygreg/python-zstandard>`__ >= 0.11.1
+-  meson >= 0.56.0
```

Comment:
We still support >=0.53.0 for regular builds, for now. It's just that our CI machine files require some >=0.56.0 features